suzieq/shared/utils.py

```diff
@@ -36,10 +35,15 @@
 MISSING_SPEED = -1
 NO_SPEED = 0
 MISSING_SPEED_IF_TYPES = ['ethernet', 'bond', 'bond_slave']
-SUPPORTED_POLLER_TRANSPORTS = ['ssh', 'https']
 SUPPORTED_ENGINES = ['pandas', 'rest']
 
 
+class PollerTransport(str, Enum):
+    """Supported poller transoport enum"""
+    ssh = 'ssh'
+    https = 'https'
```

Comment:
You need to support both http and https as transports.

Reply:
It's not what we are doing in the node as you can see [here](https://github.com/netenglabs/suzieq/blob/master/suzieq/poller/worker/nodes/node.py#L756)